buildifier2/buildifier2.go

```diff
@@ -27,16 +27,14 @@ func main() {
 
 	switch len(flag.Args()) {
 	case 0:
-		fmt.Println("Argument missing")
-		os.Exit(1)
+		log.Fatal("Argument missing")
 	case 1:
 		filename := flag.Args()[0]
 		ast, err := syntax.Parse(filename, nil, syntax.RetainComments)
 		if err != nil {
-			fmt.Printf("%+v", err)
-			os.Exit(1)
+			log.Fatalf("%+v\n", err)
 		}
-		newAst := convFile(ast)
+		newAst := convertast.ConvFile(ast)
 		fmt.Print(build.FormatString(newAst))
 
 	default:
```

Comment:
You mentioned `log.Info; os.Exit(2)` in the code review, but `log.Fatal` seems simpler. No?

Reply:
log.Fatal produces a nasty stack-trace that is no fun.  So it really shouldn't be used when Exit is available in main

Reply:
That's not what I saw when I tried it.
Looking at the doc, I see that: https://golang.org/pkg/log/#Fatal

Reply:
Ah.  I guess I am used to the internal version